rpa-home/modules/ROOT/pages/automation-userrolesandpermissions-privilegesbymodule.adoc

```diff
@@ -0,0 +1,358 @@
+= List of Privileges by Module
+
+RPA Manager has the following privileges, which can be assigned to individual users.
+
+To open any module, you always require the privilege for which the name ends in *Open*.
+
+Privileges that end with *Administration* authorize a user to create, change, and delete all types of objects in the corresponding module.
+If the authorization only applies to a single type of object in a module, the corresponding privileges end with  *Create*, *Edit*, and *Delete*.
+
+Privileges named *Global <...> View* authorize a user to see all the processes in the relevant view. A user who does not have this privilege can only see processes for which they belong to the project team.
+
+If you assign a role to a user, the user receives all the privileges contained in the role. These privileges are not displayed in the *Privileges* field in the user's profile.
+
+
+
+
+== Privileges of category *System*
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*Logfile Download*
+|Download the log files from the server using the *Download logfiles* button in the *About* dialog
+
+|*Show Event Log*
+|Display the event log of the server using the *Open Event log* button in the *About* dialog
+
+|===
+
+
+
+
+== Privileges for the *User Management* module
+
+[cols="1,2"]
+|===
+|*Privilege* |*Permitted Action*
+
+|*User Management Open*
+|Open all views of the *User Management* module.
+
+|*User Management Administration*
+|Create, change, and delete all objects of the *User Management* module.
+
+|*User Api Key Create*
+|Create user API keys
```

Comment:
This will also allow a user to create a api key for them self, without them needing the user management administration privilege

Reply:
Does a user need the User Management Administration privilege additionally to the User API Key Create privilege to be able to create API Keys for other users?

Reply:
A test showed that a user always needs the User Management Administration privilege additional to the User API Key Create privilege to create an API Key for himself or for another user. Added this to the documentation.